cmd/containerd/command/config.go

```diff
@@ -147,6 +147,16 @@ var configCommand = &cli.Command{
 					}
 				}
 
+				if config.Timeouts == nil {
```

Comment:
can we guard this section with:

```golang
if config.Version < version.ConfigVersion {
  ...
}

Reply:
Or we should add `timeout` to this `platformAgnosticDefaultConfig`.

https://github.com/containerd/containerd/blob/9b552d441582d71b7fd322c0b9f2c7c23b26b729/cmd/containerd/command/config.go#L158

Then we can eliminate the duplication/similar logic here:

https://github.com/containerd/containerd/blob/9b552d441582d71b7fd322c0b9f2c7c23b26b729/cmd/containerd/command/config.go#L59-L67

@ningmingxiao can you try if this works? (maybe try in a separate commit and if it works squash later)

Reply:
yes it works well.  https://github.com/ningmingxiao/containerd/pull/22/files